proposals/vector_to_random_points.json

```
@@ -56,7 +56,7 @@
         },
         {
             "name": "group",
-            "description": "Specifies whether the sampled points should be grouped by input geometry (default) or be generated as independent points.\n\n* If the sampled points are grouped, the process generates a `MultiPoint` per geometry given. Vector properties are preserved.\n* Otherwise, each sampled point is generated as a distinct `Point` geometry. Vector properties are *not* preserved.",
+            "description": "Specifies whether the sampled points should be grouped by input geometry (default) or be generated as independent points.\n\n* If the sampled points are grouped, the process generates a `MultiPoint` per geometry given which keeps the original identifier.\n* Otherwise, each sampled point is generated as a distinct `Point` geometry with a new identifier.",
```

What do you mean with identifier? I don't think it's a mandatory field to have.

---

```suggestion
            "description": "Specifies whether the sampled points should be grouped by input geometry (default) or be generated as independent points.\n\n* If the sampled points are grouped, the process generates a `MultiPoint` per geometry given which keeps the original identifier.\n* Otherwise, each sampled point is generated as a distinct `Point` geometry without identifier.",
```

Would that be better?

---

Yes better for both proposals!